mission-control-client/src/components/dashboard/admin-dashboard/DashboardMetrics.js

```diff
@@ -28,4 +30,7 @@ const mapStateToProps = state => {
   };
 };
 
-export default connect(mapStateToProps, null)(DashboardMetrics);
+export default connect(
+  mapStateToProps,
+  null
```

Comment:
`null` is basically an empty object, but I wonder which is considered the convention.